INSTALL.md

```diff
@@ -1,143 +1,120 @@
+
+   ```bash
+   source PATH_TO_ANACONDA/miniconda3/bin/activate
+   conda create -n NAME_OF_YOUR_PISA_ENV python=3.10
+   ```
+
+**3.** Activate the newly created environment
+
+   ```bash
+   conda activate NAME_OF_YOUR_PISA_ENV
+   ```
+
+**4.** Clone the PISA repository from github (https://github.com/icecube/pisa.git). You can also create your own fork first. For more information on how to obtain the pisa source code see [obtain-pisa-sourcecode](https://github.com/icecube/pisa/blob/master/INSTALL.md#obtain-pisa-sourcecode)
+
+   Define a directory for PISA sourcecode to live in.
+   ```bash
+   export PISA="PATH_WHERE_PISA_SHOULD_LIVE/pisa"
+   ```
+   Add this line to your `~/.bashrc` file so you can refer to the `$PISA` variable without doing this everytime. 
+   PATH_WHERE_PISA_SHOULD_LIVE could for example be the same as PATH_TO_ANACONDA.
+
+   Then clone the source code
+   ```bash
+   mkdir -p $PISA
+   git clone https://github.com/icecube/pisa.git $PISA
+   ```
+
+**5.** Install PISA with the following command
+
+   ```bash
+   pip install -e $PISA[develop] -vvv
+   ```
+
+**Explanation:**
+   * First, note that this is ***not run as administrator***. It is discouraged to do so (and has not been tested this way).
+   * `-e $PISA` (or equivalently, `--editable $PISA`): Installs from source located at `$PISA` and  allows for changes to the source code within to be immediately propagated to your Python installation.
+   Within the Python library tree, all files under `pisa` are links to your source code, so changes within your source are seen directly by the Python installation. Note that major changes to your source code (file names or directory structure changing) will require re-installation, though, for the links to be updated (see below for the command for re-installing).
+   * `[develop]` Specify optional dependency groups. You can omit any or all of these if your system does not support them or if you do not need them.
+   * `-vvv` Be maximally verbose during the install. You'll see lots of messages, including warnings that are irrelevant, but if your installation fails, it's easiest to debug if you use `-vvv`.
+   * If a specific compiler is set by the `CC` environment variable (`export CC=<path>`), it will be used; otherwise, the `cc` command will be run on the system for compiling C-code.
+
+**Note** that you can work with your installation using the usual git commands (pull, push, etc.). However, these ***won't recompile*** any of the extension (i.e. pyx, _C/C++_) libraries. See below for how to reinstall PISA when you need these to recompile.
 
-```bash
-conda create -n pisa_env python=3.10
-```
-
-3. Activate the newly created environment (You can see the name of your environment in the bash shell after the activation)
 
-```bash
-conda activate pisa_env
-```
-
-4. Clone the PISA repository from github (https://github.com/icecube/pisa.git)
-
-```bash
-git clone https://github.com/icecube/pisa.git
-```
+### Reinstall PISA
```

Comment:
I'd suggest moving the reinstallation guide to "additional information" (between "ensure a clean install ..." and "compile the documentation").